charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -15,6 +15,11 @@ podDisruptionBudget: {}
 ## Allow automount the serviceaccount token for sidecar container (eg: oauthproxy)
 automountServiceAccountToken: false
 
+## Only log messages with the given severity or above. One of: [debug, info, warn, error]
+# logLevel: info
```

Comment:
what are the defaults for these settings (when not set)?
i would prefer to have them used instead of commented out?

Reply:
Thanks for review.

It will be info.
>what are the defaults for these settings (when not set)?

Fixed. ( Also  fixed log format part same way. )
>i would prefer to have them used instead of commented out?